file/default.room.web.json

```diff
@@ -2,53 +2,43 @@
 {
 	"id":"ac2b235a-b1e2-4848-a687-6669a3f7551b",
 	"name":"Lounge",
-	"externalMeetUrl":"http://meet.google.com",
 	"disableMeeting": true
 },
 {
 	"id":"room-2",
-	"name":"CDP",
-	"externalMeetUrl":"http://meet.google.com"
+	"name":"CDP"
 },
 {
 	"id":"room-3",
-	"name":"Lorem ipsum dolor sit amet, consectetur adipiscing elit",
-	"externalMeetUrl":"http://meet.google.com"
+	"name":"Lorem ipsum dolor sit amet, consectetur adipiscing elit"
 },
 {
 	"id":"room-4",
-	"name":"Sala-4",
-	"externalMeetUrl":"http://meet.google.com"
+	"name":"Sala-4"
```

Comment:
What do you think about we change the room names for English?

Reply:
Hey @juliemar, I added some fandom references https://matrix.fandom.com/wiki/Hovercraft 😅

Reply:
so cute :)